build/config/PublishData.json

```diff
@@ -42,6 +42,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.8p1"]
         },
+       "dev15.8-preview3-vs-deps": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.9.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev15.8p1"]
```

Comment:
The "channel" here should have been `dev15.8p3`.

Reply:
Please review: https://github.com/dotnet/roslyn/pull/27574